pkg/hubble/peer/handler.go

```diff
@@ -31,12 +35,14 @@ import (
 type handler struct {
 	stop chan struct{}
 	C    chan *peerpb.ChangeNotification
+	tls  bool
 }
 
-func newHandler() *handler {
+func newHandler(withoutTLSInfo bool) *handler {
```

Comment:
I named this parameter `without...` instead of `with...` to ensure that the default value (ie false) is what we want.

> That way the way it's presented on the CLI doesn't "leak" through the codebase (if that make sense).

I don't really get this sentence. The handler is purely internal to the peer implementation (it's not even exported).

Reply:
I do understand the idea of having the zero value being the default. My point is more that while reading/writing conditions, `if !WithoutFoo` is easy to mess up.

The CLI flag is basically `true` to disable TLS, and that is mirrored by the `withoutTLSInfo` which follows the same semantic. `handler.tls` on the other hand is `false` when TLS is disabled. My point was that I would rather see consistency between the `newHandler` param and `handler.tls` letting the caller do the dance.